Day3/README.md

```diff
@@ -388,7 +388,49 @@ public class HammingDistance {
 
 }
 
+## Solution with test
```

Comment:
Add your solution under the correct section 'Java Implementation' in the README.